pkg/config/constants/constants.go

```diff
@@ -197,7 +197,7 @@ const (
 	// AmbientRedirectionDisabled is an opt-out, configured by user.
 	AmbientRedirectionDisabled = "disabled"
 
-	// AmbientUseWaypoint is the annotation used to specify which waypoint should be used for a given pod, service, etc...
+	// AmbientUseWaypoint is the label used to specify which waypoint should be used for a given pod, service, etc...
 	AmbientUseWaypoint = "istio.io/use-waypoint"
```

Comment:
nit: Consider adding Label to the name for clarity (no strongly attached to it though)

Reply:
I kinda wish we did this everywhere and we don't. Might do a followup.